### PR TITLE
Add `sdk env install` subcommand

### LINKED ISSUE #784: Feature: Add `sdk env install` subcommand
- [x] I have read the [CONTRIBUTING guidelines](CONTRIBUTING.md)

**Feature request**
As discussed some time ago in the Slack `cli-development` channel, it would be nice to have an additional subcommand for `sdk env` which installs the candidates contained in `.sdkmanrc`.

### PR DESCRIPTION
**Pull request by helpermethod**
Fixes #784.

### CODE REVIEW

**Comment by helpermethod** [comment]
Needs some more testing love 😆 .

File: src/test/resources/features/per_project_configuration.feature [hunk 1]
```diff
@@ -26,7 +26,16 @@ Feature: Per-project configuration
...
-
+		
+	Scenario: The env install subcommand is issued with an sdkman project configuration present
+		Given the system is bootstrapped
+	    And the file ".sdkmanrc" exists and contains "groovy=2.4.1"
+		And the candidate "groovy" version "2.4.1" is available for download
+		When I enter "sdk env install"
+		Then I see "Done installing!"
+		And the candidate "groovy" version "2.4.1" is installed
+		And the candidate "groovy" version "2.4.1" should not be the default
```
Comment (marc0der) [inline src/test/resources/features/per_project_configuration.feature:37]: Should we make mention that there is already a default version installed like in line 21?

**Comment by marc0der** [comment]
Looks great! Left one small comment regarding the test.

File: src/test/resources/features/per_project_configuration.feature (hunk 1 above)
Comment (helpermethod) [reply src/test/resources/features/per_project_configuration.feature:37]: Good idea!

**Comment by helpermethod** [comment]
@marc0der Ready to merge?

**Review by marc0der — APPROVED** [review]
Looks good!

**Comment by joehorsnell** [comment]
Great, glad to see this merged, it's a welcome addition - thanks!

@marc0der - do you have any plans to do a new release of SDKMAN soon? Looking at the [releases](https://github.com/sdkman/sdkman-cli/releases), 5.9.0 was in early Aug and 5.9.1 a couple of months later in early-ish Oct, so maybe a good time for 5.9.2 (he says hopefully! 😆 )?

**Pull request merged**